framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -901,3 +906,59 @@ int esm_register_user_confirm_cb(user_confirm_result_func_type func)
 		return 0;
 	}
 }
+
+int esm_save_easysetup_state(int state)
```

Comment:
The return values of esm_save_easysetup_state() and esm_read_easysetup_state() are not consistent.
In esm_save_easysetup_state(), failure with -1 and success with 0.
In  esm_read_easysetup_state(), failure with 0 and success with 1.

Reply:
Oh, you pointed out a very important part.
Thank you.
I'll fix it right away.